package.json

```diff
@@ -90,6 +93,7 @@
     "test-webgl-validation": "gulp test --webglValidation",
     "test-release": "gulp test --release",
     "generateStubs": "gulp generateStubs",
-    "sortRequires": "gulp sortRequires"
+    "sortRequires": "gulp sortRequires",
+    "uploadCesium": "gulp deploy -b cesiumjs -d cesium"
```

Comment:
As we discussed offline, rename the npm and gulp tasks to  `deploy-s3`, also don't hardcode the options here.